src/controllers/accounts/AccountsStakingPayoutsController.ts

```diff
@@ -99,9 +101,23 @@ export default class AccountsStakingPayoutsController extends AbstractController
 		{ params: { address }, query: { depth, era, unclaimedOnly, at } },
 		res,
 	): Promise<void> => {
-		const hash = await this.getHashFromAt(at);
-		const apiAt = await this.api.at(hash);
+		const earlyErasBlockInfo: IEarlyErasBlockInfo = kusamaEarlyErasBlockInfo;
+		let hash = await this.getHashFromAt(at);
+		let apiAt = await this.api.at(hash);
 		const { eraArg, currentEra } = await this.getEraAndHash(apiAt, this.verifyAndCastOr('era', era, undefined));
+		if (currentEra <= 519 && depth !== undefined) {
```

Comment:
do we intentionally throw this error for era 519 as well? (message mentions less than 518)

Reply:
Yeah, because sometimes the active era for 519 will be 518 and then sidecar will fetch the data for 517 which has the old logic and the depth will mess it up